extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -694,4 +726,99 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 }
 
+/**
+ * Keeps track of the third-party vendors' cross-domain iframes. Only one
+ * static instance will be created, which will keep and manage ALL amp-analytics
+ * tags.
+ */
+class AmpAnalyticsFrames {
+  constructor() {
+    this.requestsFrameMap = new Map();
```

Comment:
Map is not available in all environments. Please use a prototype-less object (we have a [helper](https://github.com/google/amphtml/blob/4e892b119333628b1076973da078bfc0fb07ad7b/src/utils/object.js#L20-L34)).